paf-mvp-frontend/cypress/integration/notification-widget.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { NotificationPage } from '../pages/notification.page';
+import { NotificationEnum } from '../../src/enums/notification.enum';
+
+describe('Notification widget', () => {
+  let page: NotificationPage;
+
+  context('general scenarios', () => {
+    beforeEach(() => {
+      page = new NotificationPage();
+      page.open(NotificationEnum.generalContent);
+    });
+
+    it('should exist', () => {
+      page.container.should('be.visible');
+    });
+
+    it('should be able to close snack-bar', () => {
+      page.closeBtn.click();
```

Comment:
I would add this line at the beginning of this test:
`page.container.should('be.visible');`

And remove the previous test (line 13)

This way you make sure the notification actually appears and then disappears ;-)

Reply:
good point